OTCamera/config.py

```diff
@@ -341,6 +341,7 @@ def read_text_file(text_file: Path) -> str:
 """Path to the HTML to be displayed when OTCamera is offline"""
 NUM_LOG_FILES_HTML = 2
 """Number of log files to be displayed on the status website"""
+USB_MOUNT_POINT = "~/mnt/usb"
```

Comment:
Do we need to add it to `parse_user_config()` and `user_config.example.yml`?

Reply:
In this case we don't want the user to be able to modify the mount point. The mount point needs to be the same as the entry specified in the etc/fstab which in turn is added by the install script.